cmd/app/command/device-list.go

```diff
@@ -46,3 +55,22 @@ func NewSubCmdListBlockDevice() *cobra.Command {
 
 	return getCmd
 }
+
+// deviceList prints list of devices using defaultDeviceList template
+func deviceList(kubeconfig string) {
+	ctrl, err := controller.NewController(kubeconfig)
+	if err != nil {
+		fmt.Println("error while creating controller :", err)
+		return
```

Comment:
Question - Do we not use any exit code?

Reply:
Yes it should return with exit code. my bad. added.